### PR TITLE
Part4

### PR DESCRIPTION
**Pull request by jessibcoden**
deployed:  https://allmypeeps-41395.firebaseapp.com

### CODE REVIEW

File: javascripts/controllers/DetailCtrl.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        let updateFavoriteStatus = ContactsService.createContactObject(contact);
+        ContactsService.updateContact(updateFavoriteStatus, $routeParams.contactId).then((result) => {
+            getContacts();
+        }).catch((err) => {
+            console.log("error in makeNotFavorite in DetailCtrl", err);
+        });
+    };
+
+    $scope.makeFavorite = (contact) => {
+        console.log("inside makeFavorite");
```
Comment (zoeames) [inline javascripts/controllers/DetailCtrl.js]: remove console

File: sass/main.scss [hunk 2]
```diff
@@ -19,18 +19,82 @@
 	}
```
Comment (zoeames) [inline sass/main.scss:18]: this file has a lot of extra returns.  be consistent.  no empty line before closing curly.  single line between each classs

**Comment by zoeames** [comment]
Great PR!  Just 2 minor formatting comments once those changes are in let me know and I will merge

**Comment by jessibcoden** [comment]
Removed extra returns and console log

**Comment by zoeames** [comment]
looks good 👍

**Comment by zoeames** [comment]
I apparently don't have write access so you can go ahead and merge this in

**Pull request merged**